Assets/Autopilot/Autopilot.cs

```diff
@@ -310,4 +317,24 @@ CompareTwoValues CompareValue(int valueA, int valueB)
         };
         return values;
     }
+
+    bool referenceLapDuplicated()
+    {
+        bool duplicated = false;
+        int count = 0;
+
+        for (int i = 0; i < recordedReplay.Count; i++)
+        {
+            float posX = recordedReplay[i].position.x;
+            float posZ = recordedReplay[i].position.z;
+
+            if (posX <= -661.53 && posX >= -661.73 && posZ <= 15 && posZ >= -22)
```

Comment:
Hello Jack,

If you mean the starting line with this  line, you have a trigger representing the line. The name is "StartLine 1"
Maybe you could reference the collider instead of hardcoding the values.
Moreover, you have some function helpers such has collider.bounds.Contains or collider.ClosestPoint(point) == point.
PS: Unity bounds don't have rotation, so the bounding box is usually bigger than the collider

Reply:
Hello Manuel,

I updated my pull request. Please take a look, many thanks!

Reply:
Thank you Jack. It seems ok to me